vocabularies/trs.ttl

```diff
@@ -0,0 +1,178 @@
+# baseURI: http://linked.data.gov.au/def/trs
+# imports: http://www.w3.org/2004/02/skos/core
+# imports: http://www.w3.org/2006/time
+
+@prefix trs: <http://linked.data.gov.au/def/trs#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix time: <http://www.w3.org/2006/time#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/trs>
+  a owl:Ontology ;
+  owl:imports <http://www.w3.org/2004/02/skos/core> ;
+  owl:imports <http://www.w3.org/2006/time> ;
+.
+trs:conceptScheme a skos:ConceptScheme ;
+    skos:prefLabel "Temporal Reference Systems"@en ;
+    skos:definition "Temporal reference system, such temporal coordinate systems (with origin, directions, and scales), calendar-clock combinations, or a (possibly hierarchical) ordinal system."@en ;
+    skos:historyNote "This vocabulary was established to provide instances of the Time Ontology in OWL's time:TRS class for reference. See <https://www.w3.org/TR/owl-time/#time:TRS> for the time:TRS definition."@en ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:created "2019-10-17"^^xsd:dateTime ;
+    dct:modified "2019-10-18"^^xsd:dateTime ;
+.
+trs:ChineseCalendar
+  a skos:Concept ;
+  a time:TRS ;
+  dct:source <https://aa.usno.navy.mil/faq/docs/calendars.php> ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Chinese_calendar> ;
+  skos:definition "The traditional China calendar (officially known as the Rural Calendar [農曆; 农历; Nónglì; 'farming calendar']), or Former Calendar (舊曆; 旧历; Jiùlì), Traditional Calendar (老曆; 老历; Lǎolì) or Lunar Calendar (陰曆; 阴历; Yīnlì; 'yin calendar'), is a lunisolar calendar which reckons years, months and days according to astronomical phenomena. It is defined by GB/T 33661-2017, 'Calculation and promulgation of the Chinese calendar' issued by the Standardisation Administration of China on 12 May 2017." ;
+  skos:prefLabel "Chinese Calendar" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:GPSTime
+  a skos:Concept ;
+  a time:TRS ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Global_Positioning_System#Timekeeping> ;
+  skos:definition "GPS time is not corrected to match the rotation of the Earth, so it does not contain leap seconds or other corrections that are periodically added to UTC. GPS time was set to match UTC in 1980, but has since diverged. The lack of corrections means that GPS time remains at a constant offset with International Atomic Time (TAI) (TAI − GPS = 19 seconds)" ;
```

Comment:
Ending full stop.

Reply:
Added: https://github.com/geological-survey-of-queensland/vocabularies/pull/58/commits/35ef42f4372569467da79a827ce8f9096c564189